### ISSUE TITLE
feature request: show community based rating on item

### ISSUE THREAD

**Issue by TeeTeeHaa** [joshhunt/destinySets#65]
[Destiny Item Manager](https://www.destinyitemmanager.com/) shows community based ratings for items, for example "4.9" for Origin Story. Unfortunately it shows only items you actually own. I have not yet found a website which shows those ratings for items you do not own.

Destiny Sets contains every item there is and I would really like to see the community based rating on all of them, ideally in the same format as in Destiny Item Manager (1.0 to 5.0).

**Comment by Trekoid** [comment]
The DIM ratings are collected and stored by DIM, not by Bungie. DIM would have to give @joshhunt access to that data in order to be able to display it, which doesn't currently look to be possible.

**Comment by joshhunt** [comment]
Actually, item ratings are a collab between DestinyTracker and DIM, and you can see them at the bottom of each item's page https://db.destinytracker.com/d2/en/items/3890960908

You can get there from Destiny Sets by either clicking an item, then clicking "View on Destiny Tracker" in the modal, or just right clicking an item and clicking "open in new tab".

**Comment by TeeTeeHaa** [comment]
Thank you for that information! But what is the conversion formula? For example regarding Nightshade (Pulse Rifle from Zavala):
* DIM has value of 4.3 stars (of 5 stars) based on 203 ratings (from all plattforms)
* DestinyTracker has [value of 179](https://db.destinytracker.com/d2/en/items/339163900) based on 191 positive and 12 negative ratings (sum is 203)

Assuming DIM's values go from 1 star to 4 stars my potential calculations are the following and they're all not 4.3:
* (191 / 203) * 4 + 1 = 4.8 (rounded up)
* (179 / 203) * 4 + 1 = 4.5 (rounded down)

Regarding Aeon Safe:
* DIM has value of 1 based on 55 ratings
* DestinyTracker has [value of 13](https://db.destinytracker.com/d2/en/items/1906855381) based on 34 positive and 21 negative ratings (sum is 55)

But I should ask this somewhere in DIM's community, not here.

Anyway, my personal opinion: DestinySets would have additional benefit if that value would be in the mouse-over pop-up within DestinySets. Thank you for putting it on the "distant wishlist". And thank you for making a great tool for the Destiny community.

**Comment by joshhunt** [comment]
Yeah, that's the thing - DIM does some conversion and weighting of scores
to get to their star rating. You could look through DIM github to see
exactly what they're doing, or drop them a line on Reddit or their Discord
to find out more details :)
